test/data/corner_cases/wtf?.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
(Comment related to the commit, not the line)

Adding the 3 files to `corner_cases.zim` make the zim file increasing by 32.3 KB (1300%), where remove the fulltextindex was decreasing the file by 71KB. This seems too much. Can you verify that you have commit the right zim file ?

Reply:
Everything is correct. So far `corner_cases.zim` contained only empty items which provided an opportunity to omitting the title index. Appearance of the first entry with a title resulted in the title index being generated.

Reply:
Arf. This is a pitty. I wonder if we should not have a option to not create xapian title index at all.
(Or maybe use a library compiled without xapian)

Anyway, not for this PR